bpf-map-pressure-exporter/Dockerfile

```
@@ -1,7 +1,7 @@
 # bpf-map-pressure-exporter container
 
 # Stage1: build from source
-FROM ghcr.io/cybozu/golang:1.23-jammy AS build
+FROM ghcr.io/cybozu/golang:1.24-jammy AS build
```

```suggestion
FROM ghcr.io/cybozu/golang:1.24-noble AS build
```
noble can be used :)